components/skus/browser/DEPS

```diff
@@ -1,6 +1,6 @@
 include_rules = [
   "+services/network/public",
   "+services/preferences/public",
-  "+third_party/rust/chromium_crates_io/vendor/cxx-1.0.119/include",
```

Comment:
I think these includes can be replaced with `#include "third_party/rust/cxx/v1/cxx.h"` and then we don't need to track the version here

Reply:
Updated in 42bcbd7e47dcd46458702f7abb4a7021ddf7c2fb